gatsby-config.ts

```diff
@@ -6,16 +6,28 @@ require("dotenv").config({
 
 const isAppsSite = process.env.GATSBY_INDEX_PAGE_MODE == "apps";
 
+const notaMetadata = {
+  title: "Nota",
```

Comment:
What if we set out title to something like "Nota - Event Apps"? It might be a bit more legible.